libs/components/lists/src/lib/modules/repeater/repeater-item.component.html

```diff
@@ -90,14 +94,11 @@
   <div class="sky-repeater-item-right" #itemHeaderRef>
     <div
       class="sky-repeater-item-header"
+      [attr.role]="(itemRole$ | async)?.title"
```

Comment:
Bumped the `rowheader` role to the next parent since that parent wraps the header controls, too.